### PR TITLE
SetUp Linters

### PR DESCRIPTION
**Pull request by DanielTagua**
In this branch, the following operations are performed:

- [ ] Add Html file.
- [ ] Add Css file.
- [ ] Add Readme file.
- [ ] Add linters.

### CODE REVIEW

File: .gitignore
Comment (skyv26) [inline .gitignore:1]: - [ ] Please add the `test.md` file in your project and make sure that you put the `test,md` file in the `.gitignore` to ignore the file while committing and pushing your code on the `GitHub` repo. Kindly adhere to the project requirements and add the changes and re-submit your project. Rest of the things you did a **marvelous** job 🚀 .

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @DanielTagua, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore
Comment (DanielTagua) [reply .gitignore:1]: Thank you: Do I need to do another pull request??

**Comment by DanielTagua** [comment]
I added the test.md and I put in the .getignore

**Pull request closed without merging**